tests/integration/test_model.py

```diff
@@ -223,8 +223,6 @@ async def test_deploy_local_charm_channel(event_loop):
     async with base.CleanModel() as model:
         await model.deploy(str(charm_path), channel='stable')
         assert 'charm' in model.applications
-        await model.wait_for_idle(status="active")
-        assert model.units['charm/0'].workload_status == 'active'
 
```

Comment:
shouldn't we be waiting for the unit to be idle? Or is this the bugfix you did that said "a model isn't idle until all units in the model are idle' ?

I believe that I'm ok with all the changes, except this one.